### PR TITLE
Add TestFairy and Firebase to iOS deployment docs

### PR DESCRIPTION
**Pull request by BytesGuy**
# Description
Add TestFairy and Firebase to iOS deployment docs

# Reasons
Adding another two popular deployment services to the new deployment docs.

Also removed the old deployment documentation and pointed to the new doc!

### CODE REVIEW

File: jekyll/_cci2/deploying-ios.md [hunk 1]
```diff
@@ -167,8 +241,8 @@ Once this is complete you will need to generate an API token to allow Fastlane t
 1. Go to the [API Tokens](https://appcenter.ms/settings/apitokens) section in Settings
 2. Click on "New API Token"
 3. Enter a description for the token, then set the access to "Full Access"
-
-When the token is generated, make sure to copy it somewhere safe.
+4. When the token is generated, make sure to copy it somewhere safe.
+5. GGo to your project settings in CircleCI and create a new environment variable named `VS_API_TOKEN` with the value of the API Key.
```
Comment (marcomorain) [inline jekyll/_cci2/deploying-ios.md]:
```suggestion
5. Go to your project settings in CircleCI and create a new environment variable named `VS_API_TOKEN` with the value of the API Key.
```

File: jekyll/_cci2/deploying-ios.md [hunk 2]
```diff
@@ -196,11 +270,88 @@ desc "Upload to VS App Center"
...
+
+```yaml
+# .circleci/config.yml
+version: 2.1
+jobs:
+  build:
+    #  insert build code here...
+  deploy:
+    steps:
+      - checkout
```
Comment (marcomorain) [inline jekyll/_cci2/deploying-ios.md]: you will need to use a workspace here to share the artifact from `build` to `deploy`. And you need to specify an executor in the `deploy` job.

Comment (BytesGuy) [reply jekyll/_cci2/deploying-ios.md]: @marcomorain Good point, I just grabbed this code from the previous deployment docs. Will resolve this tomorrow with a better example config

**Comment by BytesGuy** [comment]
@marcomorain For now I am just removing the curl method for TestFairy so we can expedite the changes to the docs (especially removing the old deployment section and replacing it with a link to the new doc).

**Pull request merged**